org.eclipse.swtchart.test/src/org/eclipse/swtchart/AxisTitleTest.java

```diff
@@ -177,18 +167,9 @@ public void testFont() {
 		showChart();
 		// set the disposed font
 		font.dispose();
-		try {
-			xAxisTitle.setFont(font);
-			fail();
-		} catch(IllegalArgumentException e) {
-			// expected to reach here
-		}
-		try {
-			yAxisTitle.setFont(font);
-			fail();
-		} catch(IllegalArgumentException e) {
-			// expected to reach here
-		}
+		Font font2 = font;

```

Comment:
Why is this required?

Reply:
Lambdas require that local variables used in them to be final or "effectively final". As the variable is overwritten later it is not "effectively final" while a new local variable is.